src/Classes/Utils/Constants.ts

```diff
@@ -29,6 +30,11 @@ export class Constants {
                         smooth: 10,
                     },
                 },
+                ondiv: {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'ondiv' is not sorted alphabetically](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)